include/glow/Optimizer/GraphOptimizerPipeline/Pipeline.h

```diff
@@ -123,6 +123,13 @@ class FunctionPassPipeline : private llvm::SmallVector<FunctionPassConfig, 64> {
   /// Push a new \p FPC to the end of the pipeline.
   void pushBack(FunctionPassConfig FPC) { push_back(FPC); }
 
+  /// Removes the first instance of a pass with ID \p FPID. \returns whether an
+  /// instance of the pass was successfully found and removed.
+  bool removeFirstInstanceOfPass(FunctionPassID FPID);
```

Comment:
Do you have a use-case where you want to remove only the first instance if a pass, but not other instances of the same pass?

Reply:
Nope -- I've moved this to be private.